source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+|                         | additional risk    |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+| cage_id                 | the unique         | categorical     | ``"C31"``;       |
+|                         | identifier for     |                 | ``"C35"``;       |
+|                         | each cage of       |                 | ``"C42"``;       |
+|                         | mice               |                 | ``"C43"``;       |
+|                         |                    |                 | ``"C44"``;       |
+|                         |                    |                 | ``"C49"``        |
++-------------------------+--------------------+-----------------+------------------+
+| donor                   | A unique           | categorical     | ``"hc_1"``       |
+|                         | identifier for     |                 | ``"pd_1"``       |
+|                         | the human who      |                 |                  |
+|                         | donated the        |                 |                  |
+|                         | feces              |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+| donor_status            | whether the        | categorical     | ``"Healthy"``;   |
+|                         | donor has          |                 | ``"PD"``         |
+|                         | Parkinson's        |                 |                  |
+|                         | disease or not     |                 |                  |
+|                         | (Donor             |                 |                  |
+|                         | ``pd_1`` had       |                 |                  |
+|                         | Parkinson's        |                 |                  |
+|                         | disease;           |                 |                  |
+|                         | ``hc_1``           |                 |                  |
+|                         | was                |                 |                  |
+|                         | neurologically     |                 |                  |
+|                         | healthy)           |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+| days_post_transplant    | the number of      | numeric         | 7, 14, 21, 49    |
+|                         | days after the     |                 |                  |
+|                         | mice were          |                 |                  |
+|                         | humanized          |                 |                  |
++-------------------------+--------------------+-----------------+------------------+
+
+
+Even though the mouse ID looks like a number, we will specify the type using the ``#q2_type`` column in the data.
```

Comment:
This is awesome, will be really useful in workshop situations, for discussing the column type directives.

Reply:
One of the features I struggle with often in my own analysis, since I get handed enough datasets that are numerically coded.